data/json/mapgen/station_radio.json

```
@@ -89,7 +89,8 @@
         { "group": "record_weather", "x": 18, "y": 13 },
         { "group": "cleaning", "x": 2, "y": 17, "chance": 70 },
         { "group": "electronics", "x": [ 6, 7 ], "y": 5, "chance": 80 }
-      ]
+      ],
+      "place_nested": [ { "chunks": [ [ "meteorologist_spawn", 20 ], [ "null", 50 ] ], "x": 4, "y": 13 } ]
```

So currently the chance of the meteorologist spawning is 20/70. That's a bit counterintuitive. Did you mean this?
```suggestion
      "place_nested": [ { "chunks": [ [ "meteorologist_spawn", 20 ], [ "null", 80 ] ], "x": 4, "y": 13 } ]
```
That gives them a 20% chance to spawn.

---

That was a little misunderstanding on how nested spawn chance works and I forgot to fix this. Thank you.